src/Hosting/Hosting/src/Http/DefaultHttpContextFactory.cs

```diff
@@ -49,7 +49,7 @@ public HttpContext Create(IFeatureCollection featureCollection)
         }
 
         var httpContext = new DefaultHttpContext(featureCollection);
-        Initialize(httpContext);
+        Initialize(httpContext, featureCollection);
```

Comment:
Can we add a test similar to to [InternalActiveFlagIsSetAndUnset](https://github.com/dotnet/aspnetcore/blob/2f68ef8d42646046eb36ff2db5f17965a7893545/src/Http/Http/test/DefaultHttpContextTests.cs#L259-L273) but for DefaultHttpContextFactory.Create() and DefaultHttpContextFactory.Initialize()?

Reply:
Good idea, done.